Samples/HelloDirectML/main.cpp

```diff
@@ -200,8 +204,20 @@ int main()
     dml::TensorDesc desc = { DML_TENSOR_DATA_TYPE_FLOAT32, dimensions};
     dml::Expression input = dml::InputTensor(graph, 0, desc);
 
+#if MULTIPLY_WITH_SCALAR_CONSTANT
+    // The memory referenced by any constant nodes (e.g, "scalar" below) needs to be kept alive until the graph is compiled.
+    float scalar = 3.4f;
+    auto constValue = dml::ConstantData(graph, 
```

Comment:
```suggestion
    auto constValue = dml::ConstantData(
        graph, 
```
missing new line/inconsistent indent